notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -207,9 +207,29 @@
         "if IS_WORKBENCH_NOTEBOOK:\n",
```

Comment:
<p>combine these 2 text cells</p><p> </p><p>Under Overview should be a sentence like:</p><p> </p><p>In this tutorial, you learn how ....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1051/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>